CHANGELOG.md

```diff
@@ -1,8 +1,34 @@
 ## Next release
 
+## 0.39.0 / 2020-05-06
+
+* [CHANGE] Introduce release schedule (#3135)
+* [CHANGE] Remove Custom Resource Definition management and move CRD definitions to bundle.yaml (#3155, #3171)
```

Comment:
This change removes the following options:

- manage-crds
- crd-kinds
- with-validation

Should this be described in the changelog?

Reply:
agreed, we should be very explicit even add a warning around this

Reply:
Suggestions for better entry? :)

Reply:
I would suggest adding this entry:
```[CHANGE] Remove options configuring CRD management (manage-crds, crd-kinds, with-validation) (#3155)```